### PR TITLE
Ingest COLING 2020 and workshops

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

File: data/yaml/venues.yaml [hunk 1]
```diff
@@ -248,6 +248,9 @@ discomt:
   acronym: DiscoMT
   is_acl: true
   name: Workshop on Discourse in Machine Translation
+dmr:
```
Comment (akoehn) [inline data/yaml/venues.yaml:251]: All the new venues lack an explicit `is_acl` property. I don't know whether the default is `true` or `false`, so stating it explicitly would be good imo.

**Review by danielgildea — APPROVED** [review]

**Review by akoehn — CHANGES_REQUESTED** [review]
Other than the remark, generally looks good (though I have not checked whether the W19-XX are actually correct).

File: data/yaml/venues.yaml (hunk 1 above)
Comment (mjpost) [reply data/yaml/venues.yaml:251]: I removed all `false` instances.

Comment (akoehn) [reply data/yaml/venues.yaml:237]: ah, okay. Missed that. In that case: LGTM, merge the way you like.

**Review by akoehn — APPROVED** [review]

**Comment by danielgildea** [comment]
> I forgot to add name fixing to the main ingest script.

I'm not sure exactly what the best way to automate this is.  If two names look like a match, the correct name could be the existing one or the new one.  We could ask interactively, but that could be a lot of work.  It's also not always clear which one is correct.

**Comment by nschneid** [comment]
What about sigs? I know at least SIGANN and SIGLEX have workshops.

**Comment by mjpost** [comment]
Yeah, that information is now placed in the `meta` file, so we can add it automatically. I've got to update the Anthology code to do this. I'll sneak it in tonight and then merge.

File: data/yaml/sigs/siglex.yaml [hunk 2]
```diff
@@ -2,7 +2,11 @@ Name: Special Interest Group on the Lexicon (SIGLEX)
 ShortName: SIGLEX
 URL: http://www.siglex.org/
 Meetings:
-  - 2019:
+  - 2020
+    - 2020.msr-1 # Proceedings of the Third Workshop on Multilingual Surface Realisation
```
Comment (nschneid) [inline data/yaml/sigs/siglex.yaml:6]: should be SIGGEN only?

**Pull request merged**